src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const costPerDay = 40;
```

Comment:
it's a good practice to use capital letters for hardcoded values -> `COST_PER_DAY`